Detectors/PHOS/simulation/src/GeometryParams.cxx

```diff
@@ -365,32 +361,22 @@ GeometryParams::GeometryParams(const std::string_view name)
   mNPhi = mNStripX * mNCellsXInStrip; // number of crystals across the beam
   mNz = mNStripZ * mNCellsZInStrip;   // number of crystals along the beam
 
-  // CPV Geometry
-  mCPVFrameSize[0] = 2.5;
-  mCPVFrameSize[1] = 5.1;
-  mCPVFrameSize[2] = 2.5;
-  mGassiplexChipSize[0] = 4.2;
-  mGassiplexChipSize[1] = 0.1;
-  mGassiplexChipSize[2] = 6.0;
-  mFTPosition[0] = 0.7;
-  mFTPosition[1] = 2.2;
-  mFTPosition[2] = 3.6;
-  mFTPosition[3] = 5.1;
-
-  mCPVActiveSize[0] = mNumberOfCPVPadsPhi * mCPVPadSizePhi;
-  mCPVActiveSize[1] = mNumberOfCPVPadsZ * mCPVPadSizeZ;
-  mCPVBoxSize[0] = mCPVActiveSize[0] + 2 * mCPVFrameSize[0];
-  mCPVBoxSize[1] = mCPVFrameSize[1] * mNumberOfCPVLayers + 0.1;
-  mCPVBoxSize[2] = mCPVActiveSize[1] + 2 * mCPVFrameSize[2];
-
-  mPHOSParams[0] =
-    TMath::Max((Double_t)mCPVBoxSize[0] / 2.,
-               (Double_t)(mEMCParams[0] - (mEMCParams[1] - mEMCParams[0]) * mCPVBoxSize[1] / 2 / mEMCParams[3]));
-  mPHOSParams[1] = mEMCParams[1];
-  mPHOSParams[2] = TMath::Max((Double_t)mEMCParams[2], (Double_t)mCPVBoxSize[2] / 2.);
-  mPHOSParams[3] = mEMCParams[3] + mCPVBoxSize[1] / 2.;
-
-  mIPtoUpperCPVsurface = mIPtoOuterCoverDistance - mCPVBoxSize[1];
+ 
+//  mPHOSParams[0] =
+//    TMath::Max((Double_t)mCPVBoxSize[0] / 2.,
+//               (Double_t)(mEMCParams[0] - (mEMCParams[1] - mEMCParams[0]) * mCPVBoxSize[1] / 2 / mEMCParams[3]));
+//  mPHOSParams[1] = mEMCParams[1];
+//  mPHOSParams[2] = TMath::Max((Double_t)mEMCParams[2], (Double_t)mCPVBoxSize[2] / 2.);
+//  mPHOSParams[3] = mEMCParams[3] + mCPVBoxSize[1] / 2.;
+
+   mPHOSParams[0]=78.924;
+   mPHOSParams[1]=93.704;
+   mPHOSParams[2]=80.01;
+   mPHOSParams[3]=41.91;
+
+
+   for(Int_t i=0; i<4; i++)mPHOSATBParams[i]=mPHOSParams[i]-mATBoxWall ;
```

Comment:
how about a static variable for the number of parameters and using this in the array definition and as upper bound in such loops.

Reply:
Please format the code properly using clang-format; Please add braces even around single line bodies.